src/Runtime/python/onnxmlirdocker.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
```

Comment:
That file is totally new, is that right?

Reply:
Yes, it is the file to use docker package.